articles/aks/cluster-configuration.md

```diff
@@ -150,9 +149,9 @@ kubectl get pods --all-namespaces
 
 ### Deploy an AKS Mariner cluster with an ARM template
 
-To add Mariner to an existing ARM template, you need to add `"osSKU": "mariner"` and `"mode": "System"` to `agentPoolProfiles` and set the apiVersion to 2021-03-01 or newer (`"apiVersion": "2021-03-01"`). The following deployment uses the ARM template "marineraksarm.yml".
+To add Mariner to an existing ARM template, you need to add `"osSKU": "mariner"` and `"mode": "System"` to `agentPoolProfiles` and set the apiVersion to 2021-03-01 or newer (`"apiVersion": "2021-03-01"`). The following deployment uses the ARM template "marineraksarm.json".
```

Comment:
Suggest rewording to - 

To add Mariner to an existing ARM template, you need to do the following:

* Add `"osSKU": "mariner"` and `"mode": "System"` to `agentPoolProfiles` property.
* Set the apiVersion to 2021-03-01 or newer: `"apiVersion": "2021-03-01"`

The following deployment uses the ARM template *marineraksarm.json*.